### DISCUSSION TITLE
Unable to Login/Register

### DISCUSSION THREAD

**Discussion by sillvva** [vercel/community#612] [Help] [answered]
I am trying to create an Account on Vercel, but I'm getting errors.

- When I try to sign up with email, it says email already registered. I've never registered before.
- When I try to sign up with Github, it asks for a phone number, but says it can't find the user.

Basically, no matter what I try, it's giving me an error.

I can't find any other way to contact support, so this will have to do.

**Comment by sillvva** [comment, marked as answer]
Found a support email on this discussion board. Sent an email.

**Reply by markbaseteam** [reply]
Hey I'm getting the same issue - do you mind sharing the support email/updating here on the fix?

**Reply by swarnava** [reply]
You can send it to registrations@vercel.com
